src/components/calculate/CalculateBlock.js

```diff
@@ -112,27 +112,30 @@ export default function CalculateBlock(props) {
       formValues.find((v) => v.name === 'Base_replacement_water_heater_certificates').hide = true;
     }
 
-    if (formItem.name === "F7_PDRSAug24_replacement_activity") {
```

Comment:
notes: some of the code is not consistent, ideally using eslint to fix them all but will do this slowly at later stage